receiver/httpcheckreceiver/metadata.yaml

```diff
@@ -54,3 +57,21 @@ metrics:
       monotonic: false
     unit: "{error}"
     attributes: [http.url, error.message]
+  httpcheck.match_content:
```

Comment:
match has a connotation of regular expression, I'd use the term "substring" instead of match_content.